### PR TITLE
Request for bounty payout June 2024

### PR DESCRIPTION
**Pull request by rustboyar**

### CODE REVIEW

File: src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.sol [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      );
+    }
+  }
+
+  function getBounties() public pure returns (Bounty[2] memory) {
+    return [
+      Bounty({
+        asset: AaveV3EthereumAssets.GHO_UNDERLYING,
+        recipient: 0xeD656e42612177D9c747A16604E07D5C5B031d30,
+        amount: 1_000e6
```
Comment (kyzia551) [inline src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.sol:36]:
```suggestion
        amount: 1_000e18
```
wrong number of decimals

File: src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.sol [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      }),
+      Bounty({
+        asset: AaveV3EthereumAssets.GHO_UNDERLYING,
+        recipient: 0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18,
+        amount: 100e6
```
Comment (kyzia551) [inline src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.sol:41]:
```suggestion
        amount: 100e18
```
wrong number of decimals

File: src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.t.sol [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import {AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702} from './AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702
+ * command: FOUNDRY_PROFILE=mainnet forge test --match-path=src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.t.sol -vv
+ */
+contract AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702_Test is ProtocolV3TestBase {
+  AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702 internal proposal;
+
+  uint256 TOTAL_AMOUNT = 1_100e6;
```
Comment (kyzia551) [inline src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/AaveV3Ethereum_RequestForBountyPayoutJune2024_20240702.t.sol:19]:
```suggestion
  uint256 TOTAL_AMOUNT = 1_100e18;
```
wrong number of decimals

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/RequestForBountyPayoutJune2024.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+This proposal covers that bounty, cleaning up the pipeline until today July 1st 2024.
+
+## Specification
+
+This proposal, following an approach similar to Direct-to-AIP, will go directly on-chain and will release the following funds to white-hat addresses and the Immunefi platform:
+
+$1’000 to 0xeD656e42612177D9c747A16604E07D5C5B031d30.
+
+$100 to 0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18 (immunefi.eth). This is the fee corresponding to the 10% of the previous bounty.
```
Comment (eboadom) [inline src/20240702_AaveV3Ethereum_RequestForBountyPayoutJune2024/RequestForBountyPayoutJune2024.md:23]:
```suggestion
- $1’000 to 0xeD656e42612177D9c747A16604E07D5C5B031d30.

- $100 to 0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18 (immunefi.eth). This is the fee corresponding to the 10% of the previous bounty.
```

**Review by eboadom — APPROVED** [review]

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**